src/app/results/results.component.html

```diff
@@ -1,3 +1,18 @@
+<head>
+
+<style type="text/css">
+.active {
+    color: black;
+    font-weight: bold;
+}
+</style>
+
+
```

Comment:
No need to leave lines here. Please check our code, how things work and how the indentation practices is being followed. Also, don't write `style` tag within html file. Try to write it within css file.